Formula/swi-prolog.rb

```diff
@@ -32,10 +32,18 @@ class SwiProlog < Formula
 
   def install
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args,
-                      "-DSWIPL_PACKAGES_JAVA=OFF",
-                      "-DSWIPL_PACKAGES_X=OFF",
-                      "-DCMAKE_INSTALL_PREFIX=#{libexec}"
+      args = std_cmake_args + %W[
+        -DSWIPL_PACKAGES_JAVA=OFF
+        -DSWIPL_PACKAGES_X=OFF
+        -DCMAKE_INSTALL_PREFIX=#{libexec}
+      ]
+
+      on_linux do
+        args << "-DCMAKE_C_COMPILER=/usr/bin/gcc"
+        args << "-DCMAKE_CXX_COMPILER=/usr/bin/g++"
```

Comment:
```suggestion
        args << "-DCMAKE_C_COMPILER=gcc"
        args << "-DCMAKE_CXX_COMPILER=g++"
```
Does this help with the shims audit at all?

Reply:
That did not help :(

Reply:
Yea, you'll need to patch the generated files. Try doing `brew install -si swi-prolog`, run `cmake` as in the formula, then `grep` for shim references...